src/Camelot.Services.Windows/WinApi/ShellLink.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+using System.Text;
+
+namespace Camelot.Services.Windows.ShellIcons;
```

Comment:
Should we move it to `Camelot.Services.Windows.ShellIcons`? It's icons related functionality. Or we can adjust the namespace to reflect current class location